src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutes.scala

```diff
@@ -55,9 +55,11 @@ trait GoogleExtensionRoutes extends ExtensionRoutes with UserInfoDirectives {
               }
             }
           } ~
-          path("proxyGroup") {
+          path("proxyGroup" / Segment) { targetUserEmail =>
             complete {
-              StatusCodes.OK -> googleExtensions.toProxyFromUser(userInfo.userId)
+              googleExtensions.getUserProxy(WorkbenchEmail(targetUserEmail)).map { proxyEmail =>
+                StatusCodes.OK -> proxyEmail
```

Comment:
this should return 404 if email does not exist, as it stands it will return a 500